README.md

```diff
@@ -53,6 +53,7 @@ memary will default to the locally run models unless explicitly specified.
 ```
 OPENAI_API_KEY="YOUR_API_KEY"
 NEO4J_PW="YOUR_NEO4J_PW"
+NEO4J_USER="YOUR_NEO4J_PW"
```

Comment:
I like this simple addition - added flexibility but please edit line 56 accordingly: would need to be `NEO4J_USER=YOUR_NEO4J_USERNAME`.